poi/poi_types.xml

```diff
@@ -819,7 +819,10 @@
 		</poi_category>
 		<poi_category name="finance" default_tag="finance">
 			<poi_type name="bank" tag="amenity" value="bank"/>
-			<poi_type name="atm" tag="amenity" value="atm"/>
+			<poi_filter name="atm">
```

Comment:
This was just a shot in the dark. Is there an explanation for `poi_filter` and the other tags? 

Is it because one object can only be matched once either by `poi_type` or `poi_filter`? And if you need it to be matched several times you need to use `poi_additional`?